service/metrics/dgraph_metrics.go

```diff
@@ -1,69 +1,69 @@
 package metrics
 
 import (
-        "github.com/prometheus/client_golang/prometheus"
-        "github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
did this change from space -> tab or vice versa? just make sure it's as intended and as makes `gofmt` happy

Reply:
Yes, it formatted by gofmt